tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -301,6 +301,36 @@ module.exports = {
         .waitForElementNotPresent('@loadingIndicator')
         .waitForElementPresent(this.getFileRowSelectorByFileName(element))
     },
+    assertActionDisabled: function (action, fileName) {
+      const btnSelectorHighResolution = '(' + this.getFileRowSelectorByFileName(fileName) +
+        this.elements[action + 'ButtonInFileRow'].selector + ')[1]'
+      const btnSelectorLowResolution = '(' + this.getFileRowSelectorByFileName(fileName) +
+        this.elements[action + 'ButtonInFileRow'].selector + ')[last()]'
+      const fileActionsBtnSelector = this.getFileRowSelectorByFileName(fileName) +
+        this.elements.fileActionsButtonInFileRow.selector
```

Comment:
can we put this and https://github.com/owncloud/phoenix/blob/00a9f009e7af1eacda0c6d195ee91298b4888508/tests/acceptance/pageObjects/FilesPageElement/filesList.js#L16 into one function?